### PR TITLE
Create navigation thru the app using web driver

### PR DESCRIPTION
**Pull request by ArthurNiedial**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [x] No need to
- [ ] Manually (please, describe, if necessary)
- [ ] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java [hunk 1]
```diff
@@ -64,8 +62,7 @@ void createSchemaAvro() {
     @Order(2)
     void updateSchemaAvro() {
         AVRO_API.setValuePath(System.getProperty("user.dir") + "/src/main/resources/testData/schema_avro_for_update.json");
-        mainPage.goTo()
-                .goToSideMenu(CLUSTER_NAME, MainPage.SideMenuOptions.SCHEMA_REGISTRY);
+        naviSideBar.openSideMenu(CLUSTER_NAME, NaviSideBar.SideMenuOptions.SCHEMA_REGISTRY);
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java:65]: schemaRegistryList.waitUntilScreenReady() missed

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java [hunk 2]
```diff
@@ -84,8 +81,7 @@ void updateSchemaAvro() {
     @Test
     @Order(3)
     void deleteSchemaAvro() {
-        mainPage.goTo()
-                .goToSideMenu(CLUSTER_NAME, MainPage.SideMenuOptions.SCHEMA_REGISTRY);
+        naviSideBar.openSideMenu(CLUSTER_NAME, NaviSideBar.SideMenuOptions.SCHEMA_REGISTRY);
         schemaRegistryList.openSchema(AVRO_API.getName())
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java:85]: schemaRegistryList.waitUntilScreenReady() missed

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java [hunk 3]
```diff
@@ -101,15 +97,14 @@ void deleteSchemaAvro() {
     @Order(4)
     void createSchemaJson() {
         Schema schemaJson = Schema.createSchemaJson();
-        mainPage.goTo()
-                .goToSideMenu(CLUSTER_NAME, MainPage.SideMenuOptions.SCHEMA_REGISTRY);
+        naviSideBar.openSideMenu(CLUSTER_NAME, NaviSideBar.SideMenuOptions.SCHEMA_REGISTRY);
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java:100]: schemaRegistryList.waitUntilScreenReady() missed

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java [hunk 4]
```diff
@@ -121,8 +116,7 @@ void createSchemaJson() {
     @Test
     @Order(5)
     void deleteSchemaJson() {
-        mainPage.goTo()
-                .goToSideMenu(CLUSTER_NAME, MainPage.SideMenuOptions.SCHEMA_REGISTRY);
+        naviSideBar.openSideMenu(CLUSTER_NAME, NaviSideBar.SideMenuOptions.SCHEMA_REGISTRY);
         schemaRegistryList.openSchema(JSON_API.getName())
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java:120]: schemaRegistryList.waitUntilScreenReady() missed

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java [hunk 5]
```diff
@@ -158,8 +151,7 @@ void createSchemaProtobuf() {
     @Test
     @Order(7)
     void deleteSchemaProtobuf() {
-        mainPage.goTo()
-                .goToSideMenu(CLUSTER_NAME, MainPage.SideMenuOptions.SCHEMA_REGISTRY);
+        naviSideBar.openSideMenu(CLUSTER_NAME, NaviSideBar.SideMenuOptions.SCHEMA_REGISTRY);
         schemaRegistryList.openSchema(PROTOBUF_API.getName())
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java:155]: schemaRegistryList.waitUntilScreenReady() missed

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java [hunk 6]
```diff
@@ -44,15 +44,13 @@ public void beforeAll() {
     @Test
     public void createTopic() {
         Topic topicToCreate = new Topic().setName("new-topic");
-        mainPage.goTo()
-                .goToSideMenu(CLUSTER_NAME, MainPage.SideMenuOptions.TOPICS);
+        naviSideBar.openSideMenu(CLUSTER_NAME, NaviSideBar.SideMenuOptions.TOPICS);
         topicsList.pressCreateNewTopic()
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java:48]: topicsList.waitUntilScreenReady() missed

**Review by VladSenyuta — CHANGES_REQUESTED** [review]

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java (hunk 6 above)
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java]: Added

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java (hunk 5 above)
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java]: Added

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java (hunk 4 above)
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java]: Added

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java (hunk 3 above)
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java]: Added

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java (hunk 1 above)
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java]: Added

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java [hunk 7]
```diff
@@ -42,16 +42,14 @@ public void beforeAll() {
     @Order(1)
     void createSchemaAvro() {
         Schema schemaAvro = Schema.createSchemaAvro();
-        mainPage.goTo()
-                .goToSideMenu(CLUSTER_NAME, MainPage.SideMenuOptions.SCHEMA_REGISTRY);
+        naviSideBar.openSideMenu(CLUSTER_NAME, NaviSideBar.SideMenuOptions.SCHEMA_REGISTRY);
         schemaRegistryList.clickCreateSchema()
```
[the remark this answers is not in the document]
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java]: Added

File: kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/NaviSideBar.java [hunk 8]
```diff
@@ -1,40 +1,39 @@
...
-public class MainPage {
-
-    private static final String path = "/";
+@ExtensionMethod(WaitUtils.class)
+public class NaviSideBar {
 
     @Step
-    public MainPage goTo() {
-        Selenide.open(Source.BASE_WEB_URL + path);
+    public NaviSideBar openSideMenu(String clusterName, SideMenuOptions sideMenuOptions) {
```
[the remark this answers is not in the document]
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/NaviSideBar.java]: Fixed

[the remark this answers is not in the document]
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/NaviSideBar.java]: Done.

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/ConnectorsTests.java [hunk 9]
```diff
@@ -77,14 +80,13 @@ public void createConnector() {
     @CaseId(196)
     @Test
     public void updateConnector() {
-        kafkaConnectList.goTo(CLUSTER_NAME)
-                .waitUntilScreenReady()
-                .openConnector(CONNECTOR_FOR_UPDATE.getName());
+        naviSideBar.openSideMenu(CLUSTER_NAME, NaviSideBar.SideMenuOptions.KAFKA_CONNECT);
+        kafkaConnectList.openConnector(CONNECTOR_FOR_UPDATE.getName());
```
[the remark this answers is not in the document]
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/ConnectorsTests.java]: Added

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/ConnectorsTests.java [hunk 10]
```diff
@@ -60,14 +61,16 @@ public void createConnector() {
         Connector connectorForCreate = new Connector()
                 .setName("sink_postgres_activities_e2e_checks")
                 .setConfig(getResourceAsString("config_for_create_connector.json"));
-        kafkaConnectList.goTo(CLUSTER_NAME)
+        naviSideBar.openSideMenu(CLUSTER_NAME, NaviSideBar.SideMenuOptions.KAFKA_CONNECT);
```
[the remark this answers is not in the document]
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/ConnectorsTests.java]: Fixed

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/base/BaseTest.java [hunk 11]
```diff
@@ -55,6 +62,8 @@ public void setWebDriver() {
     RemoteWebDriver remoteWebDriver = webDriverContainer.getWebDriver();
     WebDriverRunner.setWebDriver(remoteWebDriver);
     remoteWebDriver.manage().window().setSize(new Dimension(1440, 1024));
+    Selenide.open(BASE_WEB_URL);
+    naviSideBar.waitUntilScreenReady();
```
[the remark this answers is not in the document]
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/base/BaseTest.java:58]: Added

File: kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/NaviSideBar.java [hunk 12]
```diff
@@ -44,15 +43,8 @@ public enum SideMenuOptions {
...
-            clusterElement.click();
-        }
-        clusterElement
-                .parent()
-                .$x(String.format(".//ul/li[contains(.,'%s')]", option.value))
-                .click();
-        return this;
+    public void waitUntilScreenReady() {
+        $x("//*[contains(text(),'Loading')]").shouldBe(Condition.disappear);
+        $x("//h1[text()='Dashboard']").shouldBe(Condition.visible);
```
[the remark this answers is not in the document]
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/NaviSideBar.java]: Fixed

[the remark this answers is not in the document]
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/NaviSideBar.java]: Fixed

File: kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/NaviSideBar.java [hunk 13]
```diff
@@ -1,40 +1,39 @@
...
-import org.openqa.selenium.By;
 
 import static com.codeborne.selenide.Selenide.$;
 import static com.codeborne.selenide.Selenide.$x;
 
-@ExtensionMethod({WaitUtils.class})
-public class MainPage {
-
-    private static final String path = "/";
+@ExtensionMethod(WaitUtils.class)
```
[the remark this answers is not in the document]
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/NaviSideBar.java:14]: Fixed

[the remark this answers is not in the document]
Comment (VladSenyuta) [reply kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/NaviSideBar.java:14]: where??

File: kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/schema/SchemaRegistryList.java [hunk 14]
```diff
@@ -14,6 +14,13 @@ public class SchemaRegistryList {
 
     private final SelenideElement schemaButton = $(By.xpath("//*[contains(text(),'Create Schema')]"));
 
+    @Step
+    public SchemaRegistryList waitUntilScreenReady(){
+        $x("//*[contains(text(),'Loading')]").shouldBe(Condition.disappear);
+        $x("//button[text()=' Create Schema']").shouldBe(Condition.visible);
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/schema/SchemaRegistryList.java:20]: it's bad practice to use some strange spaces in locators
better use 'contains' in this case

File: kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/settings/Source.java [hunk 15]
```diff
@@ -4,4 +4,6 @@ public abstract class Source {
 
   public static String BASE_API_URL = System.getProperty("BASE_URL", "http://localhost:8080");
   public static String BASE_WEB_URL = System.getProperty("BASE_DOCKER_URL", "http://host.testcontainers.internal:8080");
+  public static final String CLUSTER_NAME = "local";
+
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/settings/Source.java:8]: extra space, remove pls

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/base/BaseTest.java (hunk 11 above)
[the remark this answers is not in the document]
Comment (VladSenyuta) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/base/BaseTest.java:58]: partially, take a look again pls

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/ConnectorsTests.java [hunk 16]
```diff
@@ -77,14 +83,18 @@ public void createConnector() {
...
-        connectorDetails.waitUntilScreenReady()
+        connectorDetails
+                .waitUntilScreenReady()
                 .openConfigTab()
                 .setConfig(CONNECTOR_FOR_UPDATE.getConfig());
-        kafkaConnectList.goTo(CLUSTER_NAME);
-        Assertions.assertTrue(kafkaConnectList.isConnectorVisible(CONNECTOR_FOR_UPDATE.getName()),"isConnectorVisible()");
+        naviSideBar
+                .openSideMenu(KAFKA_CONNECT);
+        Assertions.assertTrue(kafkaConnectList.isConnectorVisible(CONNECTOR_FOR_UPDATE.getName()), "isConnectorVisible()");
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/ConnectorsTests.java:97]: kafkaConnectList.waitUntilScreenReady() missed

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/ConnectorsTests.java [hunk 17]
```diff
@@ -93,15 +103,18 @@ public void updateConnector() {
...
                 .openConnector(CONNECTOR_FOR_DELETE.getName());
         connectorDetails
                 .waitUntilScreenReady()
                 .openDotMenu()
                 .clickDeleteButton()
                 .clickConfirmButton();
-        kafkaConnectList.goTo(CLUSTER_NAME);
+        naviSideBar
+                .openSideMenu(KAFKA_CONNECT);
         Assertions.assertFalse(kafkaConnectList.isConnectorVisible(CONNECTOR_FOR_DELETE.getName()), "isConnectorVisible()");
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/ConnectorsTests.java:118]: kafkaConnectList.waitUntilScreenReady() missed

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java [hunk 18]
```diff
@@ -42,16 +43,18 @@ public void beforeAll() {
...
                 .setSubjectName(schemaAvro.getName())
                 .setSchemaField(fileToString(schemaAvro.getValuePath()))
                 .selectSchemaTypeFromDropdown(schemaAvro.getType())
                 .clickSubmit()
                 .waitUntilScreenReady();
-        mainPage.goTo()
-                .goToSideMenu(CLUSTER_NAME, MainPage.SideMenuOptions.SCHEMA_REGISTRY);
+        naviSideBar
+                .openSideMenu(SCHEMA_REGISTRY);
         Assertions.assertTrue(schemaRegistryList.isSchemaVisible(schemaAvro.getName()),"isSchemaVisible()");
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java:58]: schemaRegistryList.waitUntilScreenReady() missed

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java [hunk 19]
```diff
@@ -64,13 +67,16 @@ void createSchemaAvro() {
...
+                .openSchema(AVRO_API.getName())
                 .waitUntilScreenReady()
                 .openEditSchema();
         Assertions.assertTrue(schemaCreateForm.isSchemaDropDownDisabled(),"isSchemaDropDownDisabled()");
-        schemaCreateForm.selectCompatibilityLevelFromDropdown(CompatibilityLevel.CompatibilityEnum.NONE)
+        schemaCreateForm
+                .selectCompatibilityLevelFromDropdown(CompatibilityLevel.CompatibilityEnum.NONE)
                 .setNewSchemaValue(fileToString(AVRO_API.getValuePath()))
                 .clickSubmit()
                 .waitUntilScreenReady();
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java:82]: insert pls schemaDetails object to understand which screen are we waiting for

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java [hunk 20]
```diff
@@ -84,9 +90,11 @@ void updateSchemaAvro() {
...
-                .goToSideMenu(CLUSTER_NAME, MainPage.SideMenuOptions.SCHEMA_REGISTRY);
-        schemaRegistryList.openSchema(AVRO_API.getName())
+        naviSideBar
+                .openSideMenu(SCHEMA_REGISTRY);
+        schemaRegistryList
+                .waitUntilScreenReady()
+                .openSchema(AVRO_API.getName())
                 .waitUntilScreenReady()
                 .removeSchema();
         Assertions.assertFalse(schemaRegistryList.isSchemaVisible(AVRO_API.getName()),"isSchemaVisible()");
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java:100]: schemaRegistryList.waitUntilScreenReady() missed

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java [hunk 21]
```diff
@@ -101,15 +109,18 @@ void deleteSchemaAvro() {
...
+                .clickCreateSchema()
                 .setSubjectName(schemaJson.getName())
                 .setSchemaField(fileToString(schemaJson.getValuePath()))
                 .selectSchemaTypeFromDropdown(schemaJson.getType())
                 .clickSubmit()
                 .waitUntilScreenReady();
-        mainPage.goToSideMenu(CLUSTER_NAME, MainPage.SideMenuOptions.SCHEMA_REGISTRY);
+        naviSideBar
+                .openSideMenu(SCHEMA_REGISTRY);
         Assertions.assertTrue(schemaRegistryList.isSchemaVisible(schemaJson.getName()),"isSchemaVisible()");
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java:124]: schemaRegistryList.waitUntilScreenReady() missed

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java [hunk 22]
```diff
@@ -121,9 +132,11 @@ void createSchemaJson() {
...
-                .goToSideMenu(CLUSTER_NAME, MainPage.SideMenuOptions.SCHEMA_REGISTRY);
-        schemaRegistryList.openSchema(JSON_API.getName())
+        naviSideBar
+                .openSideMenu(SCHEMA_REGISTRY);
+        schemaRegistryList
+                .waitUntilScreenReady()
+                .openSchema(JSON_API.getName())
                 .waitUntilScreenReady()
                 .removeSchema();
         Assertions.assertFalse(schemaRegistryList.isSchemaVisible(JSON_API.getName()),"isSchemaVisible()");
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java:142]: schemaRegistryList.waitUntilScreenReady() missed

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java [hunk 23]
```diff
@@ -138,15 +151,18 @@ void deleteSchemaJson() {
...
+                .clickCreateSchema()
                 .setSubjectName(schemaProtobuf.getName())
                 .setSchemaField(fileToString(schemaProtobuf.getValuePath()))
                 .selectSchemaTypeFromDropdown(schemaProtobuf.getType())
                 .clickSubmit()
                 .waitUntilScreenReady();
-        mainPage.goToSideMenu(CLUSTER_NAME, MainPage.SideMenuOptions.SCHEMA_REGISTRY);
+        naviSideBar
+                .openSideMenu(SCHEMA_REGISTRY);
         Assertions.assertTrue(schemaRegistryList.isSchemaVisible(schemaProtobuf.getName()),"isSchemaVisible()");
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java:166]: schemaRegistryList.waitUntilScreenReady() missed

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java [hunk 24]
```diff
@@ -158,9 +174,11 @@ void createSchemaProtobuf() {
...
-                .goToSideMenu(CLUSTER_NAME, MainPage.SideMenuOptions.SCHEMA_REGISTRY);
-        schemaRegistryList.openSchema(PROTOBUF_API.getName())
+        naviSideBar
+                .openSideMenu(SCHEMA_REGISTRY);
+        schemaRegistryList
+                .waitUntilScreenReady()
+                .openSchema(PROTOBUF_API.getName())
                 .waitUntilScreenReady()
                 .removeSchema();
         Assertions.assertFalse(schemaRegistryList.isSchemaVisible(PROTOBUF_API.getName()),"isSchemaVisible()");
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java:184]: schemaRegistryList.waitUntilScreenReady() missed

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java [hunk 25]
```diff
@@ -44,15 +45,17 @@ public void beforeAll() {
...
+                .pressCreateNewTopic()
                 .setTopicName(topicToCreate.getName())
                 .sendData()
                 .waitUntilScreenReady();
-        mainPage.goTo()
-                .goToSideMenu(CLUSTER_NAME, MainPage.SideMenuOptions.TOPICS);
-        Assertions.assertTrue(topicsList.isTopicVisible(topicToCreate.getName()),"isTopicVisible");
+        naviSideBar
+                .openSideMenu(TOPICS);
+        Assertions.assertTrue(topicsList.isTopicVisible(topicToCreate.getName()), "isTopicVisible");
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java:58]: topicsList.waitUntilScreenReady() missed

**Review by VladSenyuta — CHANGES_REQUESTED** [review]

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/base/BaseTest.java (hunk 11 above)
[the remark this answers is not in the document]
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/base/BaseTest.java:58]: Fixed

File: kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/NaviSideBar.java (hunk 13 above)
[the remark this answers is not in the document]
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/NaviSideBar.java:14]: Have done!

File: kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/schema/SchemaRegistryList.java (hunk 14 above)
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/pages/schema/SchemaRegistryList.java:20]: Fixed

File: kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/settings/Source.java (hunk 15 above)
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/settings/Source.java:8]: Fixed

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/ConnectorsTests.java (hunk 16 above)
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/ConnectorsTests.java:97]: Fixed

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/ConnectorsTests.java (hunk 17 above)
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/ConnectorsTests.java:118]: Added

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java (hunk 18 above)
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java:58]: Added

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java (hunk 19 above)
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java:82]: Added

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java (hunk 20 above)
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java:100]: Added

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java (hunk 22 above)
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java:142]: Added

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java (hunk 23 above)
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java:166]: Added

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java (hunk 24 above)
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java:184]: Added

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java (hunk 25 above)
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java:58]: Added

Comment (VladSenyuta) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java:60]: where??

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java [hunk 26]
```diff
@@ -64,17 +70,22 @@ void createSchemaAvro() {
...
-                .goToSideMenu(CLUSTER_NAME, MainPage.SideMenuOptions.SCHEMA_REGISTRY);
-        schemaRegistryList.openSchema(AVRO_API.getName())
+        naviSideBar
+                .openSideMenu(SCHEMA_REGISTRY);
+        schemaRegistryList
+                .waitUntilScreenReady()
+                .openSchema(AVRO_API.getName());
+        schemaDetails
                 .waitUntilScreenReady()
                 .openEditSchema();
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java:80]: after openEditSchema() we should wait schemaCreateForm

**Review by VladSenyuta — CHANGES_REQUESTED** [review]

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java (hunk 25 above)
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java:60]: Have done!

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java (hunk 26 above)
Comment (ArthurNiedial) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java:80]: Added

**Review by VladSenyuta — APPROVED** [review]

**Pull request merged**